packages/shared/src/utils/stateless/calendar/calendar-event/calendar-event.impl.ts

```diff
@@ -42,7 +42,8 @@ export default class CalendarEventImpl implements CalendarEventInternal {
     return (
       dateTimeStringRegex.test(this.start) &&
       dateTimeStringRegex.test(this.end) &&
-      dateFromDateTime(this.start) === dateFromDateTime(this.end)
+      dateFromDateTime(this.start) === dateFromDateTime(this.end) &&
```

Comment:
I think we should not make this change. Semantically it makes more sense if an event can be `_isSingleDayTimed`  _and_ `_isSingleMinute`. Then we can also undo the change in `packages/calendar/src/utils/stateless/events/sort-events-for-week.ts`